Rendera/src/main.cpp

```diff
@@ -4,9 +4,10 @@
 #include "camera.hpp"
 #include "sphere.hpp"
 #include "intersection.hpp"
-#include <fstream>
-#include <cmath>
-#include <random>
+#include <iostream>
```

Comment:
You don't need this header as of now. However, printing something like "PNG image generated" and "JPG image generated" at the end would be good so that the user knows that the program actually did something.